config/app.php

```diff
@@ -638,7 +638,7 @@
     | Handles page layout, login/logout, and the default settings pages. This set
     | is required.
     */
-    'modules' => explode(',', env('CYPHT_MODULES','core,contacts,local_contacts,feeds,imap,smtp,account,idle_timer,calendar,themes,nux,developer,history,saved_searches,advanced_search,highlights,profiles,inline_message,imap_folders,keyboard_shortcuts')),
+    'modules' => explode(',', env('CYPHT_MODULES','core,contacts,local_contacts,ldap_contacts,gmail_contacts,feeds,jmap,imap,smtp,account,idle_timer,desktop_notifications,calendar,themes,nux,developer,imap_folders')),
```

Comment:
Do you really want to change default modules list just to fix the testing? You'd better change the modules list confing in the testing scripts.

Reply:
I'l not change.